src/activities/geo-country/resource/board/board18_0.qml

```diff
@@ -0,0 +1,154 @@
+/* GCompris
+ *
+ * SPDX-FileCopyrightText: 2022 Jesús Espino <jespinog@gmail.com>
+ *
+ * Authors:
+ *   Jesús Espino <jespinog@gmail.com>
+ *
+ *   SPDX-License-Identifier: GPL-3.0-or-later
+ */
+import QtQuick 2.12
+
+
+QtObject {
+   property string instruction: qsTr("Regions of Spain")
+   property var levels: [
+      {
+         "pixmapfile" : "spain/spain_spain.svgz",
+         "type" : "SHAPE_BACKGROUND_IMAGE"
+      },
+      {
+          "pixmapfile": "spain/spain_andalucia.svgz",
+           //: Region of Spain: Andalucía
+           "toolTipText" : qsTr("Andalucía"),
```

Comment:
The names should be in English in all the tooltips (and then each translator translates it for its language).

Here, it should be `Andalusia` and same for the other ones. Do you want me to update them when I merge?

Reply:
However you prefer, I'm ok doing the changes myself, but whatever you think is easier for you works for me.

Reply:
If you can do it, it would be better as it would make two pairs of eyes please

Reply:
I'll do it tomorrow (i hope) unless you do it yourself before that.

Reply:
I found some time, now is everything in english and also I change the title to Autonomous communities that is exactly how are they name in Spain.